packages/core/src/__tests__/ceramic-pinning.test.ts

```diff
@@ -52,10 +53,19 @@ async function createStream(
     ceramic,
     null,
     { deterministic: true, controllers: [controller], family },
-    { anchor: false, publish: false }
+    { anchor: false, publish: false, sync: SyncOptions.NEVER_SYNC }
   )
 }
 
+async function isPinned(ceramic: CeramicApi, streamId: StreamID): Promise<boolean> {
+  const iterator = await ceramic.pin.ls(streamId)
+  const arr = []
+  for await (const id of iterator) {
+    arr.push(id)
+  }
+  return arr.length == 1 && arr[0] == streamId.toString()
+}
```

Comment:
Maybe just use `it-first`? 
https://github.com/achingbrain/it

Reply:
done